main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,44 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in the ("NEON_SVE bridge")
+<../acle.rst#_ssec-NEONSVEBridge>. These intrinics add a
```

Comment:
After a quick playaround i've not managed to get this to link to `.. _ssec-NEONSVEBridge:` at line 411 in `main/acle.rst`

Reply:
Github does not use the internal hyperlink names, but has it's own convention: https://github.com/MDevereau/acle/blob/next-release/main/acle.rst#neon-sve-bridge 

```suggestion
<../acle.rst#-neon-sve-brige>. These intrinsics add a
```

(mind a typo too)